### PR TITLE
Make improvements to resumable_bootstrap_test

### PR DESCRIPTION
**Pull request by ptnapoleon**
@mambocab to review

Some light cleanup. I was going to make the stress larger, but I no longer think the size of the bootstrap is why the test fails.

### CODE REVIEW

File: bootstrap_test.py [hunk 1]
```diff
@@ -144,7 +145,7 @@ def resumable_bootstrap_test(self):
         cluster.populate(2).start(wait_other_notice=True)
 
         node1 = cluster.nodes['node1']
-        node1.stress(['write', 'n=100K', 'cl=TWO', '-schema', 'replication(factor=2)'])
+        node1.stress(['write', 'n=100K', 'cl=TWO', '-schema', 'replication(factor=2)', '-rate', 'threads=50'])
```
Comment (mambocab) [inline bootstrap_test.py]: What's this change for? I think it's probably good, but I don't know the purpose.

Comment (ptnapoleon) [inline bootstrap_test.py]: To limit the number of threads it uses, so as not to time out, or also melt my machine. And maybe it also pins it to running once? I can't confirm.

Comment (mambocab) [inline bootstrap_test.py]: Yeah, pretty sure it pins it to a single run. And sounds good.

**Comment by mambocab** [comment]
+1

**Pull request merged**